flake.nix

```diff
@@ -5,83 +5,129 @@
     nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-unstable";
   };
 
-  outputs = { self, nixpkgs }:
-    let
-      system = "x86_64-linux";
-      version = "1.0.0-a.28";
-      downloadUrl = {
-        "specific" = {
-	  url = "https://github.com/zen-browser/desktop/releases/download/${version}/zen.linux-specific.tar.bz2";
-	  sha256 = "sha256:1vq7k3qwfdx70frng5p308zwnih86bwz75zpzcb1lbf3xsliz702";
-	};
-	"generic" = {
-	  url = "https://github.com/zen-browser/desktop/releases/download/${version}/zen.linux-generic.tar.bz2";
-	  sha256 = "sha256:154q0yl7s8v87dcpig8ixl607iqn1iv8mrlb82cldb1xn5gvlw5x";
-	};
+  outputs = {
+    self,
+    nixpkgs,
+  }: let
+    system = "x86_64-linux";
+    version = "1.0.0-a.29";
+    downloadUrl = {
+      "specific" = {
+        url = "https://github.com/zen-browser/desktop/releases/download/${version}/zen.linux-specific.tar.bz2";
+        sha256 = "sha256:0nmfqrr7x16p989m63inr9qz80l3hy94ymidn0x76j1hjldr37z5";
       };
-
-      pkgs = import nixpkgs {
-        inherit system;
+      "generic" = {
+        url = "https://github.com/zen-browser/desktop/releases/download/${version}/zen.linux-generic.tar.bz2";
+        sha256 = "sha256:0fr2h6ilfiq9jwaslq47q1hswhsdkmn9wlix7g13c1ixc7zg2f6j";
       };
+    };
+
+    pkgs = import nixpkgs {
+      inherit system;
+    };
 
-      runtimeLibs = with pkgs; [
-        libGL stdenv.cc.cc fontconfig libxkbcommon zlib freetype
-        gtk3 libxml2 dbus xcb-util-cursor alsa-lib pango atk cairo gdk-pixbuf glib
-	udev libva mesa libnotify cups pciutils
-	ffmpeg libglvnd pipewire
-      ] ++ (with pkgs.xorg; [
-        libxcb libX11 libXcursor libXrandr libXi libXext libXcomposite libXdamage
-	libXfixes libXScrnSaver
+    runtimeLibs = with pkgs;
+      [
+        libGL
+        libGLU
+        libevent
+        libffi
+        libjpeg
+        libpng
+        libstartup_notification
+        libvpx
+        libwebp
+        nasm
+        nspr
+        nss_latest
+        stdenv.cc.cc
+        fontconfig
+        libxkbcommon
+        libdrm
+        zlib
+        freetype
+        gtk3
+        libxml2
+        dbus
+        dbus-glib
+        xcb-util-cursor
+        alsa-lib
+        libpulseaudio
+        pango
+        atk
+        cairo
+        gdk-pixbuf
+        glib
+        udev
+        libva
+        mesa
+        libnotify
+        cups
+        pciutils
+        ffmpeg
+        libglvnd
+        pipewire
+      ]
+      ++ (with pkgs.xorg; [
+        libxcb
+        libX11
+        libXcursor
+        libXrandr
+        libXi
+        libXext
+        libXcomposite
+        libXdamage
+        libXfixes
+        libXScrnSaver
```

Comment:
Please, let's no introduce formatting changes in between actual changes, it makes checking what checked a lot harder than it needs to be :)

Reply:
i am so sorry alejandra is tied to my save keybind